caveclient/base.py

```diff
@@ -409,7 +409,10 @@ def wrapper(*args, **kwargs):
                 raise ServerIncompatibilityError(msg)
         if kwarg_use_constraints is not None:
             for kwarg, kwarg_constraint in kwarg_use_constraints.items():
-                if _version_fails_constraint(self.server_version, kwarg_constraint):
+                if kwargs.get(kwarg, None) is None:
```

Comment:
what about "if kwarg not in kwargs"? I think that is slightly more in line with the desired use since it is checking if the user is actually passing in the parameter?